src/containers/documentation/ApiPage.tsx

```diff
@@ -58,19 +57,17 @@ const getApi = (apiName?: string): APIDescription | null => {
 const ApiPage = (): JSX.Element => {
   const location = useLocation();
   const params = useParams<APINameParam>();
+  const flags = useFlag([FLAG_API_ENABLED_PROPERTY]);
```

Comment:
why not

```suggestion
  const flags = useFlag([FLAG_API_ENABLED_PROPERTY, api.urlFragment]);
```

(if you move it after line 62)

i would also suggest clarifying what type of flag it is in the variable name so it's easier to remember exactly which value we're using here.

Reply:
That suggestion as written wouldn't work. The available `api.urlFragment` values aren't flag names. Also if I tried to use it at the end of the `useFlag()` method then the api is possibly undefined (impossible in this routing but TypeScript doesn't know that) causing it to fallback using an inline conditional which just makes this turn ugly IMO.

```ts
const apiEnabledFlag = useFlag([FLAG_API_ENABLED_PROPERTY])[api?.urlFragment ?? ''];
```

I did end up changing the variable name to clarify things though which I think helps nicely.

Reply:
you could do this if you wanted:

```
const api = getApi(params.apiName);
const category = lookupApiCategory(params.apiCategoryKey);
// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
const apiEnabledFlag = useFlag([FLAG_API_ENABLED_PROPERTY, api!.urlFragment]);
```

but given your point, i'm fine with the current approach if that's your preference. 

also, did you push the variable change commit? don't see it right now

Reply:
Forgot to commit / push. Will do with other tweaks.